webextensions/manifest/externally_connectable.json

```diff
@@ -0,0 +1,19 @@
+{
+  "webextensions": {
+    "manifest": {
+      "externally_connectable": {
+        "__compat": {
+          "description": "Whitelist for sending messages to extension",
```

Comment:
Remove this line and replace it with <code>"mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/manifest.json/externally_connectable"</code>.